python/GafferUI/PathListingWidget.py

```diff
@@ -903,14 +936,34 @@ def updateColumnWidths( self ) :
 		self.__recalculatingColumnWidths = True
 
 		header = self.header()
-		numColumnsToResize = header.count() - 1 # leave the last section alone, as it's expandable
+		numColumnsToResize = header.count()
+		if header.stretchLastSection() or self.getStretchLastColumn() :
```

Comment:
I think we know that `stretchLastSection()` is `False` already don't we, since we hardcoded it in `_TreeView.__init__`?

Reply:
Removed in 63a7d55.